R/utilities-goodness-of-fit.R

```diff
@@ -33,8 +33,19 @@ plotMeanGoodnessOfFit <- function(structureSet,
 
   observedResult <- loadObservedDataFromSimulationSet(structureSet$simulationSet, logFolder)
 
+
+  outputSelections <- structureSet$simulationSet$outputs
+  if (!is.null(settings$outputSelections)) {
+    outputSelections <- NULL
+    for (output in structureSet$simulationSet$outputs) {
+      if (output$path %in% settings$outputSelections) {
+        outputSelections <- c(outputSelections, output)
+      }
+    }

```

Comment:
Maybe, if you want to save some if/for loops, you can try to use directly the path names with sapply
```
availableOutputs <- sapply(structureSet$simulationSet$outputs, function(output){output$path})
selectedOuptuts <- availableOutputs %in% settings$outputSelections
outputSelections <- structureSet$simulationSet$outputs[selectedOuptuts]
```

Reply:
Changed